homeassistant/components/zwave_me/config_flow.py

```diff
@@ -26,6 +26,13 @@ def __init__(self):
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user or started with zeroconf."""
         errors = {}
+        placeholders = {
+            "local_token": "/112f7a4a-0051-cc2b-3b61-1898181b9950",
+            "find_token": "0481effe8a5c6f757b455babb678dc0e764feae279/112f7a4a-0051-cc2b-3b61-1898181b9950",
+            "local_url": "192.168.1.39:8083",
+            "find_url": "wss://find.z-wave.me",
+            "remote_url": "wss://87.250.250.242:8083",
+        }
```

Comment:
What's the point of this when it's all hardcoded?

Reply:
Those are examples for the description of the integration and are used only in the description field. So nothing is hardcoded here.